source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionOverflowRenderer.mm

```diff
@@ -0,0 +1,90 @@
+//
+//  ACRActionOverflowRenderer
+//  ACRActionOverflowRenderer.mm
+//
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+#import "ACRActionOverflowRenderer.h"
+#import "ACOHostConfigPrivate.h"
+#import "ACOBaseActionElementPrivate.h"
+#import "ACRView.h"
+#import "ACRIContentHoldingView.h"
+#import "ACOActionOverflow.h"
+#import "ACRButton.h"
+#import "ACROverflowTarget.h"
+#import "ACOAdaptiveCardPrivate.h"
+#import "UtiliOS.h"
+
+@implementation ACRActionOverflowRenderer
+
++ (ACRActionOverflowRenderer *)getInstance
+{
+    static ACRActionOverflowRenderer *singletonInstance = [[self alloc] init];
+    return singletonInstance;
+}
+
+- (UIButton *)renderButton:(ACRView *)rootView
+                    inputs:(NSMutableArray *)inputs
+                 superview:(UIView<ACRIContentHoldingView> *)superview
+         baseActionElement:(ACOBaseActionElement *)acoElem
+                hostConfig:(ACOHostConfig *)acoConfig;
+{
+    if ([acoElem isKindOfClass:[ACOActionOverflow class]]) {
```

Comment:
Not sure what's the real issue here. Since I need to downcast type for `acoElem` from `ACOBaseActionElement` to its sub class `ACOActionOverflow` so OO-wised using RTTI is the most safe approach. 

But anyway, changed to what you suggested.